### PR TITLE
Cmrt enhancement

### PR DESCRIPTION
**Pull request by MidhunKodiyath**
Cmrt enhancement related to cache and other changes.

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -3,6 +3,7 @@ intel Linux graphics/media team working on the CMRT:
 Daniel Charles
 Sean V Kelley
 Sameer Kibey
+Midhun Kodiyath
```
Comment (chivakker) [inline AUTHORS]: Preferably do not change the authors file for a patch set

Comment (chivakker) [inline AUTHORS]: @MidhunKodiyath due to all patches in patch set include the proper sign-off already

File: src/cm_device.cpp [hunk 2]
```diff
@@ -1581,6 +1581,46 @@ INT CmDevice::GetGenStepInfo(UINT platform, char *&stepinfostr)
...
+            upper_bound = HSW_L3_CONFIG_NUM;
+            l3_c = HSW_L3_PLANE;
+            break;
+
+        case IGFX_GEN8_CORE:
+            upper_bound = BDW_L3_CONFIG_NUM;
+            l3_c = BDW_L3_PLANE;
+            break;
+
+         default:
```
Comment (wlin70) [inline src/cm_device.cpp]: Skylake/Gen9 was already supported (pushed by Lina) , please add L3# configuration for this platform.

**Comment by MidhunKodiyath** [comment]
Can we merge these changes and move on or do we need to run QA cycle before the merge?
As I mentioned earlier this is not enabled in SKL.Tested on SKL and VP9 hybrid dec is not broken because of these changes.

I will be submit another patch soon to enable this in SKL.

**Comment by seanvk** [comment]
Please rebase and fix SKL support.  Please remove the Authors patch.

**Comment by MidhunKodiyath** [comment]
Done..!Also added this feature for SKL.

**Comment by seanvk** [comment]
Merged, thanks.

**Pull request merged**